app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_res = {}
```

Comment:
Please, use whole words as a variable names. It will make them more self-descriptive